test/functionalTest/cases/0979_patch_v2_entities_eid/patch_v2_entities_eid.test

```diff
@@ -168,14 +168,14 @@ Date: REGEX(.*)
 06. PATCH /v2/entities/E1 attr3=6, to see the error (attr3 doesn't exist)
 =========================================================================
 HTTP/1.1 404 Not Found
-Content-Length: 79
+Content-Length: 106
 Content-Type: application/json
 Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Date: REGEX(.*)
 
 {
-    "description": "The entity does not have such an attribute",
-    "error": "NotFound"
+    "description": "one or more of the attributes in the request do not exist: attr3",
```

Comment:
I should be:

```suggestion
    "description": "one or more of the attributes in the request do not exist: [ attr3 ]",
```

as I understand there could be a comma-separate list of attributes (that are better printed as array, wich `[...]`)